src/pycolmap/scene/database.cc

```diff
@@ -100,6 +454,22 @@ void BindDatabase(py::module& m) {
              return std::make_pair(std::move(all_pair_ids),
                                    std::move(all_matches));
            })
+      .def(
+          "read_num_matches",
+          [](const Database& self) {
+            std::vector<std::pair<image_pair_t, int>> pair_ids_and_num_matches =
+                self.ReadNumMatches();
+            std::vector<image_pair_t> all_pair_ids;
+            all_pair_ids.reserve(pair_ids_and_num_matches.size());
+            std::vector<int> all_num_matches;
+            all_num_matches.reserve(pair_ids_and_num_matches.size());
+            for (auto& [pair_id, num_matches] : pair_ids_and_num_matches) {
+              all_pair_ids.push_back(pair_id);
+              all_num_matches.push_back(num_matches);
+            }
+            return std::make_pair(std::move(all_pair_ids),
+                                  std::move(all_num_matches));
+          })
```

Comment:
This does not seem to be called from any file or test. It was added here just for completeness of the interface, right?